Blish HUD/Controls/Textbox.cs

```diff
@@ -1,233 +1,136 @@
 ﻿using System;
-using System.Drawing;
-using System.Windows.Forms;
 using Blish_HUD.Input;
 using Microsoft.Xna.Framework;
 using Microsoft.Xna.Framework.Graphics;
-using MonoGame.Extended.BitmapFonts;
-using Color = Microsoft.Xna.Framework.Color;
-using MouseEventArgs = Blish_HUD.Input.MouseEventArgs;
-using Point = Microsoft.Xna.Framework.Point;
-using Rectangle = Microsoft.Xna.Framework.Rectangle;
+using Microsoft.Xna.Framework.Input;
 
 namespace Blish_HUD.Controls {
+    public class TextBox : TextInputBase {
 
-    /// <summary>
-    /// Represents a textbox control.
-    /// </summary>
-    public class TextBox : Control {
+        private const int STANDARD_CONTROLWIDTH  = 250;
+        private const int STANDARD_CONTROLHEIGHT = 27;
 
-        #region Load Static
+        private const int TEXT_HORIZONTALPADDING = 10;
 
-        private static MouseInterceptor           _sharedInterceptor;
-        private static System.Windows.Forms.Label _sharedUnfocusLabel;
+        #region Load Static
 
         private static readonly Texture2D _textureTextbox;
 
         static TextBox() {
```

Comment:
Static contructor still needed?

Reply:
For the purpose of maintaining consistency with the rest of the project, yes.  I dislike assigning textures inline.

Reply:
Can be resolved then